Makefile

```diff
@@ -22,13 +22,21 @@ help: Makefile
 	@sed -n 's/^##//p' $< | column -t -s ':' |  sed -e 's/^/ /'
 .PHONY: help
 
+setup-solidity-ibc-eureka:
```

Comment:
I would prefer to call it something like install-dependencies

Reply:
https://github.com/celestiaorg/celestia-zkevm-ibc-demo/pull/38/commits/a709328586612f3f74aea241a6ff99997aa0a0c1